### PR TITLE
Random songs & star song toggle

### PR DESCRIPTION
**Pull request by 474r4x14**
Three main changes:

- Press 's' to add random songs to the queue
- Songs with a Star status now displays a little red heart next to the title
- Press 'y' on a song to toggle the star

### CODE REVIEW

**Comment by wildeyedskies** [comment]
nice!

File: gui.go [hunk 1]
```diff
@@ -537,11 +647,15 @@ func (ui *Ui) createQueuePage(titleFlex *tview.Flex) *tview.Flex {
 	queueFlex := tview.NewFlex().SetDirection(tview.FlexRow).
 		AddItem(titleFlex, 1, 0, false).
 		AddItem(ui.queueList, 0, 1, true)
-
+// TODO add like action here
```
Comment (wildeyedskies) [inline gui.go:650]: didn't you set `y` as the like action?

**Review by wildeyedskies — COMMENTED** [review]
This looks good, only one nit about the TODO. Also would you mind updating the README?

File: gui.go (hunk 1 above)
Comment (474r4x14) [reply gui.go]: My bad, forgot to remove the TODO

**Comment by 474r4x14** [comment]
I've updated the README & did some minor tidying (including the removal of the old TODO)

**Comment by wildeyedskies** [comment]
Looks great, thanks!

**Pull request merged**